app/views/admin/redirections/index.html.erb

```diff
@@ -1,36 +1,62 @@
-<div class="header-row">
-  <div class="redirection">
-    <div>Slug</div>
-    <div class="url">URL</div>
-    <div class="original-url">Original URL</div>
-    <div class="edit"></div>
-    <div class="block-button"></div>
-    <div class="unlink-button"></div>
-  </div>
-</div>
+<h2>Sort by</h2>
+<section class="sort">
+  <%= link_to(
+    "Ring order",
+    admin_redirections_path(sort_key: "next_id", sort_dir: "desc")
+  ) %>
+  <%= link_to(
+    "Last seen",
+    admin_redirections_path(sort_key: "last_seen_at", sort_dir: "desc")
+  ) %>
+  <%= link_to(
+    "Where'd you go?",
+    admin_redirections_path(sort_key: "last_seen_at", sort_dir: "asc")
+  ) %>
+</section>
 
-<% @redirections.each do |redirection| %>
-  <div class="redirection" data-id="<%= redirection.id %>">
-    <div>
-      <label>Slug</label>
-      <span class="content"><%= redirection.slug %></span>
-    </div>
-    <div class="url">
-      <label>URL</label>
-      <span class="content"><%= link_to redirection.url, redirection.url %></span>
-    </div>
-    <div class="original-url">
-      <label>Original URL</label>
-      <%= link_to redirection.original_url, redirection.original_url %>
-    </div>
-    <div class="edit">
-      <%= link_to "Edit", edit_admin_redirection_path(redirection) %>
-    </div>
-    <div class="block-button">
-      <%= button_to "Block and unlink", admin_redirection_blocks_path(redirection) %>
-    </div>
-    <div class="unlink-button">
-      <%= button_to "Unlink", admin_redirection_unlinks_path(redirection) %>
+<h2>Redirections</h2>
+<section class="redirections">
+  <div class="header-row">
+    <div class="redirection">
+      <div class="slug">Slug</div>
```

Comment:
Since it's a grid, does this need to have the classes? Does it need to be nested in a `.redirection`? Could it be this?

```erb
<section class="redirections">
  <div class="header-row">
    <% ["Slug", "URL", "Original URL", "Last seen", "", "", ""].each do |column| %>
      <div><%= column %></div>
    <% end %>
  </div>
<div>
```

Reply:
You're totally right it doesn't need the classes. I don't love the loop, though, especially with the empty strings. I'll just leave them their literal `div`s.